nmrfx-core/src/main/java/org/nmrfx/chemistry/constraints/AngleConstraint.java

```diff
@@ -335,13 +335,7 @@ public String toSTARString() {
 //      _Torsion_angle_constraint.Angle_upper_bound_val
         result.append(String.format("%.0f", Math.toDegrees(getUpper())));
         result.append(sep);
-
-        String ssID = null;
-        if (ssID == null) {
-            result.append(".");
-        } else {
-            result.append(ssID);
-        }
+        result.append(".");  // ssID
```

Comment:
set to always use '.' like in CoordinateSTARWriter.java (https://github.com/nanalysis/nmrfx/blob/93169b5a59d91aec1c532cd09140b3f3c873ffb9/nmrfx-core/src/main/java/org/nmrfx/chemistry/io/CoordinateSTARWriter.java#L100) and Noe.java(https://github.com/nanalysis/nmrfx/blob/7d9b8e15b120fd9ddb133501a913dcd9801e67fe/nmrfx-core/src/main/java/org/nmrfx/chemistry/constraints/Noe.java#L499) Although there is a note with NOE about fixing something...